pkg/pillar/types/global.go

```diff
@@ -943,6 +955,15 @@ func validateSyslogKernelLevel(level string) error {
 	return nil
 }
 
+func validateCustomResolution(res string) error {
+	// it can be notset or one of the predefined resolutions
+	if res == FmlResolutionNotSet || res == FmlResolution1024x768 || res == FmlResolution1920x1080 {
```

Comment:
What is limiting the set of available resolutions? Qemu or our code?

Reply:
Probably we need to create custom OVMF variable (a binary that we can't generate on the fly) with predefined resolution and load it via qemu, so it needs to be one of the values we already created OVMF_vars for.

There is the `-device VGA,xres=1280,yres=800` option too, but I'm not sure if that works, but if later works, then I will remove the limitation.